app/models/manageiq/providers/workflows/automation_manager/configuration_script_source.rb

```diff
@@ -5,6 +5,18 @@ def self.display_name(number = 1)
     n_('Repository (Embedded Workflows)', 'Repositories (Embedded Workflows)', number)
   end
 
+  def self.seed
+    manager = ManageIQ::Providers::Workflows::AutomationManager.in_my_region.first
+    return if manager.nil?
+
+    manager.configuration_script_sources.find_or_create_by!(:type => name, :name => "Embedded Workflows")
+  end
+
+  def git_repository
+    return super unless name == "Embedded Workflows"
```

Comment:
Let's move the "internal" name into a constant so it's shared with the create side.